src/modules/controllers/task.controller.js

```diff
@@ -0,0 +1,64 @@
+const Task = require('../../db/models/task/index');
```

Comment:
no need to create so many folders, the db folder is not needed and task too, name the file with the "task" model instead of "index" at once